Lab5.py

```diff
@@ -0,0 +1,87 @@
+class Flower:
+    def __init__(self, name, height, size, color, price, quantity, delivery_rate):
+        self.name = name
+        self.height = height
+        self.size = size
+        self.color = color
+        self.price = price
+        self.quantity = quantity
+        self.delivery_rate = delivery_rate
+
+
+class FlowerShop:
+    def __init__(self):
+        self.stock = []
+
+    def add_flower(self, flower, quantity):
+        self.stock.extend([flower] * quantity)
+
+    def remove_flower(self, flower):
+        if flower in self.stock:
+            self.stock.remove(flower)
+
+    def get_the_most_expensive_flowers(self, n=1):
+        sorted_flowers = sorted(
+            self.stock, key=lambda flower: flower.price, reverse=True
+        )
+        return sorted_flowers[:n]
+
+
+class Bouquet:
+    def __init__(self):
+        self.flowers = []
+
+    def add_flower(self, flower, quantity=1):
+        self.flowers.extend([flower] * quantity)
+
+    def get_the_most_expensive_flower(self, n=1):
```

Comment:
recommendation
change 'get_the_most_expensive_flower' to 'get_the_most_expensive_flowers'